WORKSPACE

```diff
@@ -1,18 +1,66 @@
 workspace(name = "rules_detekt")
 
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
 
 # Runtime
 
 ## Dependencies
 
-load("//detekt:dependencies.bzl", "rules_detekt_dependencies")
+### Java
```

Comment:
Huh. Yeah, it will work (I think) but rule users will be forced to declare all dependencies on their own. This is easy to reproduce with creating a completely separate workspace and attempting to use `rules_detekt` there. It works there of course and will work at the Lyft repo since all dependencies are declared there (_I can only guess as a legal Lyft employee_). However, at least `rules_proto` is far less common than `rules_java`.

In fact, the whole motivation behind #36 was to provide a way to use `rules_detekt` without declaring everything that is needed internally by the rule itself.

Reply:
Yeah, this is a bit lost on me as my understanding of included WORKSPACE is that it'll inflate their dependencies even if I don't have them in my WORKSPACE, I'll need to play with it a bit more…

Reply:
Sadly when you depend on an external repo, Bazel doesn't parse its `WORKSPACE` at all. So with those changes, like @arturdryomov mentioned, clients would have to replicate the set of dependencies.

The `rules_kotlin` repo is moving away from this problem (partly, that's not the intent afaik, just a side effect) by shipping a prebuilt worker (bazelbuild/rules_kotlin#271), maybe a similar solution could help here?